samples/HostedInAspNet.Client/HostedInAspNet.Client.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+<Project Sdk="Microsoft.NET.Sdk.Razor/2.1.0-preview2-30159">
```

Comment:
This is the way to bring in an SDK from a .nupkg.

Thoughts on this as a temporary solution? This requires a 2.1 CLI, but not a particular one as long as it's as new as preview1 (I think)